aws/data_source_aws_codecommit_repository_test.go

```diff
@@ -0,0 +1,45 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSCodeCommitRepositoryDataSource_basic(t *testing.T) {
+	rName := fmt.Sprintf("tf-acctest-%d", acctest.RandInt())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckAwsCodeCommitRepositoryDataSourceConfig(rName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.aws_codecommit_repository.default", "repository_name", rName),
+					resource.TestMatchResourceAttr("data.aws_codecommit_repository.default", "arn",
```

Comment:
Can we switch these to use [`resource.TestCheckResourceAttrPair()`](https://godoc.org/github.com/hashicorp/terraform/helper/resource#TestCheckModuleResourceAttrPair) where possible? This helps us ensure that the data source definitely matched the correct resource. Thanks! 👍